client/landing/stepper/declarative-flow/internals/steps-repository/goals/goals.tsx

```diff
@@ -32,5 +32,7 @@ export const useGoals = (): Goal[] => {
 			key: SiteGoal.Other,
 			title: translate( 'Other' ),
 		},
-	].filter( ( { key } ) => key !== Onboard.SiteGoal.Import );
+	]
+		.filter( ( { key } ) => key !== Onboard.SiteGoal.Import )
```

Comment:
I would define the following at the top of the file, outside the component:
```
const { SiteGoal } = Onboard;
const HIDE_GOALS = [ SiteGoal.Import, SiteGoal.Goal ];
````
then I would filter the array just once as follow:
`.filter( ( { key } ) => ! HIDE_GOALS.includes(key) )`